### ISSUE TITLE
FTop Value Won’t Work

### ISSUE THREAD

**Issue by LiTMACiD1** [novucs/factions-top#114]
Hi there every time I try to add value of spawners for ftop worth they don’t work only a few work help as I really need it sorted by today 🤞

**Comment by Phoenix616** [comment]
Please provide some more information like

- Your plugin version
- Your server version
- Your config

**Comment by LiTMACiD1** [comment]
Can u add me on Discord?

**Comment by Phoenix616** [comment]
> Can u add me on Discord?

No. Please provide the information here so that all developers have access to it.

**Comment by LiTMACiD1** [comment]
There 
Server Version 1.8.9
Ftop plugin version 1.2.0
Config: pastebin.com/WufdQRFb

**Comment by LiTMACiD1** [comment]
You got a fix for my problem? :)

**Comment by Phoenix616** [comment]
Can you please try a build of the latest code and see if that changes anything. (E.g. from my build server: https://ci.minebench.de/job/FactionsTop/)

**Comment by LiTMACiD1** [comment]
Yeah I’ll try that now can I use the same config for it tho

**Comment by LiTMACiD1** [comment]
Do I need to change the version of minecraft server

**Comment by Phoenix616** [comment]
No, just download build 5 from the linked page and try that.

**Comment by Phoenix616** [comment]
Also make sure that you are using the actual EntityType names for the spawner entities that you want to match (it should print errors in your log if invalid types are found). See [here](https://jd.andross.fr/spigot-1.8.8-R0.1-SNAPSHOT/javadoc/org/bukkit/entity/EntityType.html) for a list of entities that were available on your old server version.

**Comment by LiTMACiD1** [comment]
Doesn’t work still can u make changes to my config file so that it will work with all my prices

**Comment by Phoenix616** [comment]
> can u make changes to my config file so that it will work with all my prices

Just use the linked site to fix it with the correct entity names.
